sys-boot/limine/limine-9.2.2.recipe

```diff
@@ -6,7 +6,7 @@ SOURCE_URI="https://github.com/limine-bootloader/limine/releases/download/v$port
 REVISION="1"
 LICENSE="BSD (2-clause)"
 COPYRIGHT="2019-2025 mintsuki and contributors"
-CHECKSUM_SHA256="bd18ba20473b14c2cded73431dbf7dcecf7d327aff6dedaac66cf263477e1528"
+CHECKSUM_SHA256="47460e776a45bd4f54f14b7a6f6c1302d06b2df668b4a13033fdb0b2c8e4a9e0"
```

Comment:
could you please reorder a bit:
HOMEPAGE
COPYRIGHT
LICENSE
REVISION
SOURCE_URI
CHECKSUM_SHA256

Thanks!

Reply:
Should be done

Reply:
Actually SUMMARY and DESCRIPTION were schon ok

Reply:
Done